tendrl/ceph_integration/objects/rbd/atoms/rbd_not_exists/__init__.py

```diff
@@ -26,15 +26,18 @@ def run(self):
                 cluster_id=NS.tendrl_context.integration_id,
             )
         )
-        try:
-            NS.etcd_orm.client.read(
-                'clusters/%s/Pools/%s/Rbds/%s' % (
-                    NS.tendrl_context.integration_id,
-                    self.parameters['Rbd.pool_id'],
-                    self.parameters['Rbd.name']
+        if self.parameters['Rbd.pool_id'] is not None:
+            try:
+                NS.etcd_orm.client.read(
```

Comment:
Use Tendrl's common etcd orm instead of raw reads 

To check existence of an object, use https://github.com/Tendrl/commons/blob/develop/tendrl/commons/objects/__init__.py#L120